### PR TITLE
5 get articles queries

### PR DESCRIPTION
**Pull request by LukeHarrisonDev**

### CODE REVIEW

File: __tests__/server.test.js [hunk 1]
```diff
@@ -38,21 +49,65 @@ describe("/api/topics", () => {
...
+                body.articles.forEach((article) => {
+                    expect(article).toHaveProperty("author")
+                    expect(article).toHaveProperty("title")
+                    expect(article).toHaveProperty("article_id")
+                    expect(article).toHaveProperty("topic")
+                    expect(article).toHaveProperty("created_at")
+                    expect(article).toHaveProperty("votes")
+                    expect(article).toHaveProperty("article_img_url")
+                    expect(article).toHaveProperty("comment_count")
+                })
```
Comment (Harrison365) [inline __tests__/server.test.js:90]: Good. Please also check their value types.

File: models/articles.models.js [hunk 2]
```diff
@@ -5,14 +5,26 @@ function fetchArticleById(id) {
...
 
+function fetchArticles() {
+    let sqlString = 
+    `SELECT articles.author, title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url, COUNT(comments.article_id) AS comment_count
+    FROM articles
+    LEFT JOIN comments ON articles.article_id = comments.article_id
+    GROUP BY articles.article_id
+    ORDER BY created_at DESC`
+    
+    return db.query(sqlString)
```
Comment (Harrison365) [inline models/articles.models.js:24]: Nice 😎

File: __tests__/server.test.js (hunk 1 above)
Comment (LukeHarrisonDev) [reply __tests__/server.test.js:90]: I'm on it, cheers :D

File: models/articles.models.js (hunk 2 above)
Comment (LukeHarrisonDev) [reply models/articles.models.js:24]: Cheers :)

**Pull request merged**